ui/app/[locale]/(authorized)/kafka/[kafkaId]/consumer-groups/[groupId]/reset-offset/ResetOffset.tsx

```diff
@@ -179,21 +179,32 @@ export function ResetOffset({
             <FormSection title={t("offset_details")}>
               <FormGroup label={t("new_offset")}>
                 <SelectComponent<OffsetValue>
-                  options={[
-                    { value: "custom", label: t("offset.custom") },
-                    { value: "latest", label: t("offset.latest") },
-                    { value: "earliest", label: t("offset.earliest") },
-                    {
-                      value: "specificDateTime",
-                      label: t("offset.specific_date_time"),
-                    },
-                  ]}
+                  options={
+                    selectTopic === "allTopics"
```

Comment:
I think the specific offset should also be disabled for "All Partitions".

```suggestion
                    selectTopic === "allTopics" || selectPartition === "allPartitions"
```

Side note: reset/dry run button remain disabled when entering an offset of `0`. Probably due to `0` being "falsy" when setting the `isEnabled` boolean.

Reply:
fixed both the issue